### PR TITLE
[DOCS] Clarify alert de-duplication logic around configured 'additional look-back time'

### PR DESCRIPTION
**Pull request by joepeeples**
Addresses #1030.

Previews:
- [Create a detection rule | Set the rule's schedule](https://security-docs_1125.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#rule-schedule): Original location of the note, with some minor edits.
- [Monitor and Troubleshoot Rule Executions | Troubleshoot gaps](https://security-docs_1125.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-monitor.html#troubleshoot-gaps): Additional location for the same info, formatted as body text for flow and to avoid note overload.

### CODE REVIEW

**Review by spong — APPROVED** [review]
LGTM! 👍 Two in one day -- thanks @joepeeples! 🙂

File: docs/detections/rules-ui-monitor.asciidoc [hunk 1]
```diff
@@ -36,6 +36,13 @@ If you see values in the Gaps column in the All rules table or on the Rule detai
 for a small number of rules, you can increase those rules'
 Additional look-back time (*Detect* -> *Rules* -> the rule's *All actions* button (*...*) -> *Edit rule settings* -> *Schedule* -> *Additional look-back time*).
 
+It is recommended to set the `Additional look-back time` to at
```
Comment (nastasha-solomon) [inline docs/detections/rules-ui-monitor.asciidoc:39]:
```suggestion
It's recommended to set the `Additional look-back time` to at
```

File: docs/detections/rules-ui-monitor.asciidoc [hunk 2]
```diff
@@ -36,6 +36,13 @@ If you see values in the Gaps column in the All rules table or on the Rule detai
...
+least 1 minute. This ensures there are no missing alerts when a rule does not
```
Comment (nastasha-solomon) [inline docs/detections/rules-ui-monitor.asciidoc:40]:
```suggestion
least 1 minute. This ensures there are no missing alerts when a rule doesn't
```

**Review by nastasha-solomon — APPROVED** [review]
Two super minor suggestions to use contractions for a less formal tone, but otherwise LGTM! Also, I very much prefer using italics to boldface when emphasizing a word or phrase, so I'm completely onboard with that change! :)

**Review by jmikell821 — APPROVED** [review]
LGTM

**Pull request merged**